airbyte_cdk/sources/declarative/schema/default_schema_loader.py

```diff
@@ -37,7 +37,7 @@ def get_json_schema(self) -> Mapping[str, Any]:
 
         try:
             return self.default_loader.get_json_schema()
-        except OSError:
+        except (OSError, ValueError):
```

Comment:
Outside of the scope of this PR. See https://airbytehq-team.slack.com/archives/C02U9R3AF37/p1743795253674379 for more details